src/app/page.js

```diff
@@ -1,9 +1,9 @@
-import Landing from "@/components/home/Landing";
+import Button from "@/components/Button";
 
 const home = () => {
   return (
     <div>
-      <Landing />
+      <Button />
```

Comment:
it's ok to have ur button in here for testing but make sure to revert the file to its original condition before committing your changes